src/pegout/components/PegoutForm.vue

```diff
@@ -11,49 +11,53 @@
     <v-row>
       <v-col>
         <rbtc-input-amount
-          @getQuotes="getQuotes"
+          @valid-amount="checkValidAmount"
           :clear="clearAmount" />
       </v-col>
     </v-row>
-    <template v-if="showStep && !loadingQuotes">
+    <template v-if="showStep">
       <v-row>
         <span class="font-weight-bold ma-4 mb-2
         ">Select Mode</span>
       </v-row>
       <v-row>
-        <v-col cols="6" v-if="!flyoverResponded" >
-        <pegout-option class="mr-2" flyover-not-available
-          :option-type="pegoutType.FLYOVER">
-        <template v-slot>
-          <h4>
-            <span class="text-orange">Fast Mode</span> is unavailable at this time.
-          </h4>
-        </template>
-        </pegout-option>
-      </v-col>
-      <v-col cols="6" v-for="(quote, index) in pegoutQuotes" :key="index" >
-        <pegout-option
-          :option-type="pegoutType.FLYOVER"
-          :class="index === 0 ? 'mr-2' : 'ml-2'"
-          :quote="quote"
-          :isWalletAuthorizedToSign="isWalletAuthorizedToSign"
-          @openAddressDialog="showAddressDialog = true"
-          @changeSelectedOption="changeSelectedOption"
-          :selectedOption="selectedOption === quote.quoteHash"
-          :quote-difference="quoteDifference"
-          />
-      </v-col>
-      <v-col cols="6">
-        <pegout-option class="ml-2"
-          :option-type="pegoutType.POWPEG"
-          :quote="nativeQuote"
-          :isWalletAuthorizedToSign="isWalletAuthorizedToSign"
-          @openAddressDialog="showAddressDialog = true"
-          @changeSelectedOption="changeSelectedOption"
-          :selectedOption="selectedOption === ''"
-          :quote-differences="quoteDifference"
-          />
-      </v-col>
+        <v-col cols="6" v-if="!flyoverIsEnabled" >
+          <pegout-option class="mr-2" flyover-not-available
+            :option-type="pegoutType.FLYOVER">
+            <template v-slot>
+              <h4 v-if="countdown === 30">
```

Comment:
Please include the value 30 into a variable. QQ This number can be gt 30?

Reply:
Ok i can inclided in a variable, and as far as I now it only y¿takes 30 seconds to be able to regenerate the token, so it should be bigger than 30